src/set_timeout.py

```diff
@@ -31,8 +31,8 @@ def __init__(self, configs, args):
 
     def _update_pending_child(self, parent_id):
         """Set child node status to timeout when parent is timed out"""
-        child_nodes = self._db.get_nodes({"status": "pending",
-                                          "parent": parent_id})
+        child_nodes = (self._db.get_nodes({"status": "pending",
+                                          "parent": parent_id})).get('items')
```

Comment:
Maybe the `.get('items')` part should be done in `Database.get_node()` in kernelci-core instead?

Reply:
Yes, we can do that. However, the client will not receive other information from the response. The typical response from pagination is a dictionary with items (node objects), total (number of nodes received), limit, and offset values.

Reply:
In that case we may add something to the Database class methods to handle that, but I think exposing the raw data returned from the API is a bit fragile.  Say, if we change the API implementation then the code using the Database class will have to be updated too.  Using it as an abstraction layer helps with keeping the client code compatible.

Maybe an initial step would be to keep the `Database.get_nodes()` method return the same kind of data as before, using `.items()`.  Then we can look into how to properly handle pagination in this class.